docs/sql-guide/data-types/data-type-timestamp.md

```diff
@@ -8,12 +8,12 @@ nav_order: 8
 
 # TIMESTAMP data type
 
-Timestamp is a date-time data type used with `timeunit` and `epoch` constraints.
+Timestamp is a date-time data type used with `timeunit` constraint. Timestamp columns store date-time as integer value representing number of timeunits elapsed since [Unix epoch](https://www.unixtutorial.org/unix-epoch/)
```

Comment:
How about:

"...Timestamp is a date-time data type which represents the specified `timeunit` as integer values elapsed since the specified Unix Epoch..."

I'd also add a "Before you begin" heading which would include the unix epoch link:

```
## Before you begin
* [Learn about unix epoch/unix time](https://en.wikipedia.org/wiki/Unix_time){:target="_blank"
```